yeahcheese/app/Http/Controllers/EventController.php

```diff
@@ -66,14 +67,23 @@ public function store(Request $request)
     // TODO: resquestに認証キー
     public function show(Request $request)
     {
-        $event = Event::where('auth_key', $request->auth_key)->first();
-        if(is_null($event))
-        {
-            $pictures = null;
+        $today = CarbonImmutable::now()->toDateString();
+
+        $event = Event::AuthKeyEquals($request->auth_key)
+            ->ReleaseDateBefore($today)
+            ->EndDateAfter($today)
```

Comment:
scope 使う側ではキャメルケースにしよう
```suggestion
        $event = Event::authKeyEquals($request->auth_key)
            ->releaseDateBefore($today)
            ->endDateAfter($today)
```